src/cairo/cairo_renderer.cpp

```diff
@@ -255,14 +255,7 @@ struct cairo_render_marker_visitor
                 marker_tr *= tr_;
             }
             marker_tr *= agg::trans_affine_scaling(common_.scale_factor_);
-
-            //Since the underlying type is a pod_array that creates a copy (memcpy) version
-            //of the attributes, using a copy here would mean that some underlying types
-            //using ref counts (e.g shared_ptr's) would have an extra reference but the
-            //count wouldn't be increased. This would mean that we'd try to free them an
-            //extra time. To avoid this we pass the raw pointer and remove the deleter
-            svg_attribute_ptr attributes(&vmarker->attributes(), [](svg_attribute_type*){});
-
+            auto const& attributes = vmarker->attributes();
```

Comment:
Re. the removed comment: personally I try not to change code in the surroundings in order to ease updating from upstream, PR's to upstream etc. Not a big deal though.

Reply:
The comment above was mine from a previous PR. Since we aren't using neither shared_ptr nor pod_array it's no longer an issue